src/lib/deprecations.js

```diff
@@ -0,0 +1,66 @@
+const path = require('path')
+const process = require('process')
+
+const pathExists = require('path-exists')
+const semverLessThan = require('semver/functions/lt')
+
+const { NETLIFYDEVWARN } = require('../utils/logo')
+
+const warnOnOldNodeVersion = ({ log, chalk }) => {
+  if (semverLessThan(process.version, '10.0.0')) {
+    log(
+      `${NETLIFYDEVWARN} ${chalk.bold('Netlify CLI')} will require ${chalk.magenta.bold(
+        'Node.js 10',
+      )} or greater soon. Please update your Node.js version.`,
+    )
+  }
+}
+
+const NETLIFY_DIR = 'netlify'
+const DEFAULT_FUNCTIONS_SRC = path.join(NETLIFY_DIR, 'functions')
+
+const formatDirectory = (chalk, directory) => chalk.redBright(directory)
+
+const getCommunityLink = (chalk) =>
+  chalk.magenta(
+    'community.netlify.com/t/upcoming-change-netlify-functions-as-zero-config-default-folder-for-deploying-netlify-functions/28789',
```

Comment:
Are we intentionally leaving out `https://`?

Reply:
The purpose was to keep the link shorter (and the browser redirects to `https`).
But you're right, better to add it.